content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -283,6 +272,21 @@ There are some implicit conventions worth noting here:
 
     {{< codenew file="pods/topology-spread-constraints/one-constraint-with-nodeaffinity.yaml" >}}
 
+The scheduler doesn't have prior knowledge of all the zones or other topology domains that a cluster has. They are determined from the existing nodes in the cluster. This could lead to a problem in autoscaled clusters, when a node pool (or node group) is scaled to zero nodes and the user is expecting them to scale up, because, in this case, those topology domains won't be considered until there is at least one node in them.
+
+### Other Conventions
```

Comment:
Can we say `Semantics` instead of `Conventions`? @Huang-Wei ?

Reply:
SG. How about "Other Noticeable Semantics"

Reply:
notable